src/libsync/account.cpp

```diff
@@ -50,6 +58,12 @@ Account::Account(QObject *parent)
     , _credentialManager(new CredentialManager(this))
 {
     qRegisterMetaType<AccountPtr>("AccountPtr");
+
+    _resourcesCache = new ResourcesCache(this);
+    // TODO: use temporary directory? not really necessary with the current setup
```

Comment:
You use the url and the etag as cache keys, if the etag changes the file will remain for all eternity.
I'd suggest to use a temporary directory, the files are already persisted using the network cache.

Reply:
We decided on using a temporary directory within the user's cache directory, preferably next to the network cache directory.